recipes/busybox/files/busybox-mdev

```diff
@@ -36,8 +36,12 @@ fi
 
 #mdev_dash_s SECTION_START
 # Create tmpfs for /dev
-echo "Creating tmpfs at /dev"
-mount -t tmpfs tmpfs /dev -o size=64k,mode=0755
+if mount | grep -q 'devtmpfs on /dev type devtmpfs'; then
```

Comment:
You could also use `mountpoint` command, but we unfortunately have it disabled in default busybox config :disappointed: